### PR TITLE
DOC-2103: document new `border` property for the iframe dialog component

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2103: document new `border` property for the iframe dialog component

Changes:
* Added file, `dialog_border.adoc`, to `/partials/configuration/`.
* Documentation and code-sample added to file.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] Files has been included where required (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/dialog_border.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[[border]]
+=== `border`
+
+include::partial$misc/admon-requires-6.6v.adoc[]
+
+`+border+` is an optional property, available to add to the `iframe` dialog component.
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/dialog_border.adoc:6]:
```suggestion
`+border+` is an optional property, available to add to the `iframe-preview` dialog component.
```

File: modules/ROOT/partials/configuration/dialog_border.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The property has a default value of `+false+`.
+
+When this property is set to true, a border displays around the `iframe` component.
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/dialog_border.adoc:10]:
```suggestion
When this property is set to true, a border displays around the `iframe-preview` component.
```

File: modules/ROOT/partials/configuration/dialog_border.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+The `iframe` component is also highlighted when it takes focus (by, for example, using the Tab key to change focus within the active dialog).
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/dialog_border.adoc:12]:
```suggestion
The `iframe-preview` component is also highlighted when it takes focus (by, for example, using the Tab key to change focus within the active dialog).
```

File: modules/ROOT/partials/configuration/dialog_border.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+*Type:* `+Boolean+`
+
+*Possible values:* `+true+`, `+false+`
+
+*Default value:* `+false+`
+
+=== Example: using `border` to display a border around an `iframe` component
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/dialog_border.adoc:20]:
```suggestion
=== Example: using `border` to display a border around an `iframe-preview` component
```

**Review by kemister85 — COMMENTED** [review]
@brian-forte I think we should refer to the `iframe` as `iframe-preview` as this seems to be more commonly referred to as in other documentation, such as Tech-Brief. 

@MitchCTiny maybe you can confirm here.

File: modules/ROOT/partials/configuration/dialog_border.adoc (hunk 1 above)
Comment (MitchCTiny) [reply modules/ROOT/partials/configuration/dialog_border.adoc:6]: The component is just `iframe`

**Comment by MitchCTiny** [comment]
> @brian-forte I think we should refer to the `iframe` as `iframe-preview` as this seems to be more commonly referred to as in other documentation, such as Tech-Brief.
> 
> @MitchCTiny maybe you can confirm here.

The component itself is called `'iframe'`, I've just been referring to the iframe component as it's used in the AI plugin as the `preview` component.

**Review by MitchCTiny — APPROVED** [review]

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/partials/configuration/dialog_border.adoc (hunk 2 above)
Comment (mkzhx) [inline modules/ROOT/partials/configuration/dialog_border.adoc:10]:
```suggestion
When this property is set to `true`, a border displays around the `iframe` component.
```

File: modules/ROOT/partials/configuration/dialog_border.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+----
+tinymce.activeEditor.windowManager.open({
+  title: 'Dialog Title', // The dialog's title - displayed in the dialog header
+  body: {
+    type: 'panel', // The root body type - a Panel or TabPanel
+    items: [ // A list of panel components
+      {
+        name: 'preview',
+        type: 'iframe',
+        border: 'true',
```
Comment (mkzhx) [inline modules/ROOT/partials/configuration/dialog_border.adoc:32]:
```suggestion
        border: true,
```

**Review by mkzhx — APPROVED** [review]

File: modules/ROOT/partials/configuration/dialog_border.adoc (hunk 3 above)
Comment (mkzhx) [inline modules/ROOT/partials/configuration/dialog_border.adoc:12]:
```suggestion
The border is also highlighted when the `iframe` component takes focus (by, for example, using the Tab key to change focus within the active dialog).
```

**Review by mkzhx — APPROVED** [review]

**Review by ArvinJ-H — APPROVED** [review]

**Pull request merged**